Add the remote IP address to the "you can't relay" error message.

This patch is distributed under the same terms as qmail-ldap.

This patch is no longer maintained.

Author: John Morrissey <jwm@horde.net>

--- qmail-1.03.stock/qmail-smtpd.c.orig	Thu Jan 10 14:18:37 2002
+++ qmail-1.03/qmail-smtpd.c	Thu Jan 10 14:18:11 2002
@@ -139,7 +139,7 @@
 
 void err_bmf() { out("553 syntax error, please forward to your postmaster (#5.7.1)\r\n"); }
 void err_maxrcpt() { out("553 sorry, too many recipients (#5.7.1)\r\n"); }
-void err_nogateway() { out("553 sorry, that domain isn't in my list of allowed rcpthosts (#5.7.1)\r\n"); }
+void err_nogateway() { out("553 sorry, that domain isn't in my list of allowed rcpthosts for "); out(remoteip); out(" (#5.7.1)\r\n"); }
 void err_badbounce() { out("550 sorry, I don't accept bounce messages with more than one recipient. Go read RFC2821. (#5.7.1)\r\n"); }
 #ifdef TLS
 void err_nogwcert() { out("553 no valid cert for gatewaying (#5.7.1)\r\n"); }
